lib/broccoli/ember-app.js

```diff
@@ -335,7 +335,7 @@ EmberApp.prototype._initVendorFiles = function() {
   }
 
   // don't crash if ember-cli-shims is not included
-  if (!bowerDeps['ember-cli-shims']) {
+  if (!ember && !bowerDeps['ember-cli-shims']) {
```

Comment:
In other words, what's the story of this: https://github.com/ember-cli/ember-cli/pull/6254#issuecomment-260717239

It's distinct from the `ember-resolver` issue but in that PR.